src/Sulu/Bundle/TestBundle/Testing/KernelTestCase.php

```diff
@@ -76,7 +76,9 @@ protected function getKernel(array $options = [])
             'sulu_context' => 'admin',
         ], $this->getKernelConfiguration(), $options);
 
-        $kernel = new \AppKernel(
+        $kernelClass = getenv('KERNEL_CLASS') ?: '\AppKernel';
```

Comment:
Is the `KERNEL_CLASS` also what Symfony is doing now? Resp. is that required for the symfony bridge?

Reply:
its required when we update to the WebTestCase which extends the [KernelTestCase](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php#L44)